test/integration/userStatus.test.js

```diff
@@ -125,45 +159,44 @@ describe("UserStatus", function () {
         });
     });
 
-    it("Should return 401 for unauthorized request", function (done) {
+    it("Should update the User Status when requested by Super User", function (done) {
```

Comment:
shouldn't users be able to change their ooo state why do we need a superuser to do so?

Reply:
a user can update their user status while a super user can update anyone's user status.